code/datums/diseases/advance/symptoms/voice_change.dm

```diff
@@ -37,12 +37,19 @@ Bonus
 				if(ishuman(M))
 					var/mob/living/carbon/human/H = M
 					var/random_name = ""
-					switch(H.gender)
-						if(MALE)
-							random_name = pick(first_names_male)
-						else
-							random_name = pick(first_names_female)
-					random_name += " [pick(last_names)]"
+					if(H.dna.species.id == "lizard")

```

Comment:
I don't like these checks everywhere.

Write a proc to obtain a random name from the species datum and just have it be
```random_name = H.dna.species.pick_random_name(H.gender)```